### PR TITLE
Added shortcut to rename files

### LINKED ISSUE #17756: Add configurable shortcut for "Rename.." file on "FileSystem" editor. [Feature request]
**Godot version:** v3.0.2 stable official

**OS/device including version:** Windows 10

**Issue description:**

Add configurable shorkey for "Rename.." file on "FileSystem" editor. [Feature request]

image

There is currently no shortcut. 
For windows users is common to rename "things" (files, folders, etc) with the F2 key. I don't know for other OS. 
The F2 key is currently used as a shortkey for the "3D" mode. So it could be enough to add the shortkey as "None". 

image

### PR DESCRIPTION
**Pull request by Rubonnek**
Closes #17756

### CODE REVIEW

File: editor/filesystem_dock.cpp [hunk 1]
```diff
@@ -1770,6 +1772,7 @@ FileSystemDock::FileSystemDock(EditorNode *p_editor) {
 	ED_SHORTCUT("filesystem_dock/copy_path", TTR("Copy Path"), KEY_MASK_CMD | KEY_C);
 	ED_SHORTCUT("filesystem_dock/duplicate", TTR("Duplicate..."), KEY_MASK_CMD | KEY_D);
 	ED_SHORTCUT("filesystem_dock/delete", TTR("Delete"), KEY_DELETE);
+	ED_SHORTCUT("filesystem_dock/rename", TTR("Rename"), FOCUS_NONE);
```
Comment (groud) [inline editor/filesystem_dock.cpp]: I'm not sure "FOCUS_NONE" is a valid value for that argument.
If the shortcut is wanted blank you can just remove the last argument.

**Review by groud — CHANGES_REQUESTED** [review]

File: editor/filesystem_dock.cpp (hunk 1 above)
Comment (Rubonnek) [reply editor/filesystem_dock.cpp]: @groud Thanks! I was not sure what I should put there. It's fixed now.

**Review by groud — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Now that Godot 3.1 has been released, we don't plan to cherry-pick new features and enhancements to the 3.0 branch, unless there is very strong support in favor of it. Removing cherrypick label for 3.0.

**Pull request merged**